### PR TITLE
Docs, part 9

### PR DESCRIPTION
**Pull request by arogachev**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | #358

### CODE REVIEW

File: src/PostValidationHookInterface.php [hunk 1]
```diff
@@ -5,11 +5,19 @@
 namespace Yiisoft\Validator;
 
 /**
- * Allows implementing post-validation processing in the data set object itself.
- *
- * @link https://github.com/yiisoft/form/blob/d2333f9a0a77f6dcb00db1ab8ee95ec4426ea133/src/FormModel.php#L202
+ * An optional interface for data set ({@see DataSetInterface}) to implement (used as a replacement). It provides
```
Comment (samdark) [inline src/PostValidationHookInterface.php:8]: Used as a replacement of what?

**Review by samdark — APPROVED** [review]

File: src/Helper/SkipOnEmptyNormalizer.php [hunk 2]
```diff
@@ -8,13 +8,32 @@
 use Yiisoft\Validator\EmptyCriteria\NeverEmpty;
 use Yiisoft\Validator\EmptyCriteria\WhenEmpty;
```
Comment (vjik) [inline src/Helper/SkipOnEmptyNormalizer.php:10]:
```suggestion
```

File: src/PostValidationHookInterface.php (hunk 1 above)
Comment (arogachev) [reply src/PostValidationHookInterface.php:8]: `DatasetInterface` + `PostValidationHookInterface` - addition, wrong.
`DataSetInterface` -> `PostValidationHookInterface` - replacement, right.`

Comment (arogachev) [reply src/PostValidationHookInterface.php]: @samdark suggested to remove it completely.

**Pull request merged**